census_data_downloader/base.py

```diff
@@ -96,6 +112,8 @@ def download_nationwide(self):
         """
         Download nationwide data.
         """
+        if "nationwide" not in self.GEO_LIST:
+            raise NotImplementedError(f"Data only available for these geographies: {', '.join(self.GEO_LIST)}")
```

Comment:
Do we truly need this check? What use case do you have in mind here?

Reply:
I added them so that each download method is passable as a public API -- at least in parts of a project I was working on, I looped over a whole bunch of tables and download a handful of geographies for each, so it's handy to be able to handle an exception when one's not available. 

I think it also makes for easier debugging, and makes the library slightly more user-friendly for people who don't know the nuances of census geography or what's is/isn't available. Still, if you'd prefer we take these out, I'm happy to do so.

Reply:
Okay. I still think there's another step we can take in the future to streamline the different downloader functions, but we can put that beyond the scope of this ticket.